src/storage/src/hummock/compactor/mod.rs

```diff
@@ -475,6 +475,8 @@ impl Compactor {
         let mut system =
             System::new_with_specifics(RefreshKind::new().with_cpu(CpuRefreshKind::everything()));
         let pid = sysinfo::get_current_pid().unwrap();
+        let mut pending_pull_task_count: u32 = 0;
+        let running_task_count = Arc::new(AtomicU32::new(0));
```

Comment:
Suggest storing this variable in `CompactorContext` so that we can read it in other module